app/controllers/concerns/threatmetrix_review_concern.rb

```diff
@@ -11,6 +11,6 @@ def redirect_to_threatmetrix_review
 
   def threatmetrix_review_pending?
```

Comment:
Looks like this method is only used in a few places. Do you want to update the name to `fraud_review_pending?` to match the other language changes?

Reply:
I agree with changing to `fraud_review_pending?`